core/roboconf-dm/src/main/java/net/roboconf/dm/management/Manager.java

```diff
@@ -156,6 +163,14 @@ public void start() {
 		this.timer.scheduleAtFixedRate( new CheckerHeartbeatsTask( this ), 0, Constants.HEARTBEAT_PERIOD );
 
 		reconfigure();
+
+		// Starts listening to the debug queue.
+		try {
+			this.messagingClient.listenToDebugMessages( ListenerCommand.START );
```

Comment:
This should be done in reconfigure.
The DM can indeed start with a wrong messaging configuration. This configuration can be updated at runtime and then become valid. It is at this moment we should listen to debug messages.